stripe_official.php

```diff
@@ -1333,59 +1333,44 @@ public function cleanModuleCache()
     public static function getWebhookUrl()
     {
         $context = Context::getContext();
-        $id_lang = self::getLangIdContext();
         $id_shop = self::getShopIdContext();
 
         return $context->link->getModuleLink(
             'stripe_official',
             'webhook',
             [],
             true,
-            $id_lang,
+            null,
```

Comment:
I'm not sure but you may introduce a regression of older version of PrestaShop...

Reply:
We don't use $id_lang in webhook process and it may cause problem in some case